src/dat/view.cljs

```diff
@@ -266,14 +266,14 @@
 (representation/register-representation
   ::pull-summary-string
   (fn [_ _ pull-data]
-    ;[:span
+    [:span
```

Comment:
It's not currently using that, it's using a plain query. Should it be using ::pull-summary-string?